style/NewStyles.xaml

```diff
@@ -0,0 +1,10 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <!-- Add Controls for Previewer Here -->
+    </Border>
+  </Design.PreviewWith>
```

Comment:
I dont think we need this?? this is a style and shouldnt need any previewing i dont think.

Reply:
It's a special feature that allows you to see the control you are writing styles for.

Reply:
Interesting... Never knew that was a thing lol.